common/psbt_open.c

```diff
@@ -60,7 +60,7 @@ static int compare_outputs_at(const struct output_set *a,
 static const u8 *linearize_input(const tal_t *ctx,
 				 const struct wally_psbt_input *in)
 {
-	struct wally_psbt *psbt = create_psbt(NULL, 1, 0, 0);
+	struct wally_psbt *psbt = create_psbt(tmpctx, 1, 0, 0);
```

Comment:
nack, this is in common and tmpctx isn't passed in? maybe we do this other places but this binds the code to the existence/use of tmpctx as a global in all places...

Reply:
Mh, ok maybe I am missing some semantics around `tmpctx`? Why the `tmpctx` is not right here and also in other places?